### PR TITLE
Immigrate test mysql scenario.

### PR DESCRIPTION
**Pull request by wayilau**
Please answer these questions before submitting pull request

- Why submit this pull request?
- [ ] Bug fix
- [ ] New feature provided
- [ ] Improve performance
- [x] Tests cases.

### CODE REVIEW

**Comment by wu-sheng** [comment]
@wayilau Send a mail to your cmss mailbox, please check.

**Comment by wayilau** [comment]
> @wayilau Send a mail to your cmss mailbox, please check.

@wu-sheng  I received message you just wrote.

File: test/plugin/scenarios/mysql-scenario/config/expectedData.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+registryItems:
+  applications:
+    - {mysql-scenario: nq 0}
+  instances:
+    - {mysql-scenario: 1}
+  operationNames:
+    - mysql-scenario: [/mysql-scenario/case/mysql]
```
Comment (dmsolr) [inline test/plugin/scenarios/mysql-scenario/config/expectedData.yaml]: Are you missing some operatioName? For example, /JDBI/statement/execute.

File: Jenkinsfile-Agent-Test-3 [hunk 2]
```diff
@@ -67,7 +67,7 @@ pipeline {
                 sh './mvnw -f test/plugin/pom.xml clean package -DskipTests docker:build'
             }
         }
-        stage('Test Cases Report (161)') {
+        stage('Test Cases Report (214)') {
```
Comment (dmsolr) [inline Jenkinsfile-Agent-Test-3]: Could you make it place at Workload2-Group2?

Comment (wayilau) [reply Jenkinsfile-Agent-Test-3]: I am glad to.

**Comment by arugal** [comment]
Ref #3583

**Comment by arugal** [comment]
Hi, I have a question, what do we not use ```JVM-container```？

File: test/plugin/scenarios/mysql-scenario/config/expectedData.yaml (hunk 1 above)
Comment (wayilau) [reply test/plugin/scenarios/mysql-scenario/config/expectedData.yaml]: I will copy it from original source.

**Comment by wayilau** [comment]
> Hi, I have a question, what do we not use `JVM-container`？

original tests cases use tomcat, so i follow. @arugal

**Comment by wu-sheng** [comment]
Jenkins jobs have been disabled, please add your tasks to GitHub action file, and fix the conflict, remove the Jenkins file. https://github.com/apache/skywalking/blob/master/.github/workflows/plugins-test.yaml

**Comment by wayilau** [comment]
> Jenkins jobs have been disabled, please add your tasks to GitHub action file, and fix the conflict, remove the Jenkins file. https://github.com/apache/skywalking/blob/master/.github/workflows/plugins-test.yaml

ok.

**Comment by wayilau** [comment]
> > Hi, I have a question, what do we not use `JVM-container`？
> 
> original tests cases use tomcat, so i follow. @arugal

As  I have moved to JVM container. Beacause tomcat met some problems.

**Comment by dmsolr** [comment]
It is very sad to tell you that you need to update the Action configuration (resolve conflicts) because those are updated after #3956 merged. Maybe you need to move it to another group which is faster.
And the test failed (Tomcat-container doesn't need to pack it as a ZIP package. )

**Comment by wayilau** [comment]
> It is very sad to tell you that you need to update the Action configuration (resolve conflicts) because those are updated after #3956 merged. Maybe you need to move it to another group which is faster.
> And the test failed (Tomcat-container doesn't need to pack it as a ZIP package. )

ok, I am using jVM container. May be i lost some files. i will check then submit again.

**Comment by wu-sheng** [comment]
Jenkins is not there any more. We are working out with Apache why those are still there.

**Comment by wu-sheng** [comment]
You should add MySQL As a separated group in that test case, named As MySQL case like others.

**Comment by wayilau** [comment]
> You should add MySQL As a separated group in that test case, named As MySQL case like others.

ok

**Comment by wu-sheng** [comment]
What is `PluginsTest / .github/workflows/plugins-test.yaml (pull_request)`?

File: .github/workflows/plugins-test.yaml [hunk 3]
```diff
@@ -464,4 +464,30 @@ jobs:
       - name: Build the Docker image
         run: ./mvnw -f test/plugin/pom.xml clean package -DskipTests docker:build -DBUILD_NO=local >/dev/null
       - name: Run kafka 0.11.0.0-2.3.0 (16)
-        run: bash test/plugin/run.sh kafka-scenario
\ No newline at end of file
+        run: bash test/plugin/run.sh kafka-scenario
+
+Mysql:
```
Comment (wu-sheng) [inline .github/workflows/plugins-test.yaml]: This is not a correct format, This is a YAML.

Comment (kezhenxu94) [reply .github/workflows/plugins-test.yaml]: The indentation is not correct IMO, syntax looks good

**Comment by dmsolr** [comment]
It still fails. Please recheck.

**Comment by wayilau** [comment]
> What is `PluginsTest / .github/workflows/plugins-test.yaml (pull_request)`?
Log tells me that docker startup failed. I don't know why. Could you help me check ? @dmsolr

**Comment by wu-sheng** [comment]
@wayilau I think it was caused by the wrong YAML format. Has been fixed. Now, you need to make your tests passed.

**Comment by wayilau** [comment]
> @wayilau I think it was caused by the wrong YAML format. Has been fixed. Now, you need to make your tests passed.

I think  i know why. I missed some commits.

**Comment by dmsolr** [comment]
It looks like good.
Let's wait for this result. :)

**Comment by wu-sheng** [comment]
@wayilau  I don't get any reply from you about the mail. Are you planning to do?

**Comment by wayilau** [comment]
> @wayilau I don't get any reply from you about the mail. Are you planning to do?

@wu-sheng 
I did not receive any emails  from you yet. So i don't understand ..

**Comment by wu-sheng** [comment]
Could you send a mail to me? I send you a mail based on your GitHub homepage mail, maybe it is incorrect? Mind is correct, you could send a mail to there.

**Comment by wayilau** [comment]
> Could you send a mail to me? I send you a mail based on your GitHub homepage mail, maybe it is incorrect? Mind is correct, you could send a mail to there.

I am sorry, i just saw mail from my  rubbish mail box. May be the mail settings is limit  by the commany.  I will read the mail first. @wu-sheng

**Pull request merged**